mongo/src/test/java/org/axonframework/extensions/mongo/eventsourcing/eventstore/MongoOptionsFactoryTest.java

```diff
@@ -16,9 +16,11 @@
 
 package org.axonframework.extensions.mongo.eventsourcing.eventstore;
 
-import com.mongodb.MongoClientOptions;
+import com.mongodb.MongoClientSettings;
 import org.junit.jupiter.api.*;
 
+import java.util.concurrent.TimeUnit;
+
 import static org.junit.jupiter.api.Assertions.*;
 
 /**
```

Comment:
We should rename this class to `MongoSettingsFactoryTest` to be inline with the object it tests.